src/main/resources/org/traccar/web/client/i18n/Messages_pl.properties

```diff
@@ -433,3 +434,9 @@ copyPermissionsFrom = Skopiuj uprawnienia z
 confirmation = Potwierdzenie
 actionNotReversible = Czy jesteś pewny?\n Ta akcja jest nieodwracalna.
 userGroupUsersTitle = Użytkownicy grupy "{0}"
+# Audit dialog
+auditLogEvent = Wydarzenie
+auditLogAgent = Wykonawca
+show = POKAŻ
+auditInfo = Log audytu zawiera informacje o zmianach dotyczących bezpieczeństwa i uprawnień
+showLogFrom = Pokaż wpisy od
```

Comment:
W jaki sposób są tłumaczony napis <DATE> **do** <DATE>?

Reply:
klucz to z wartością do już był

Reply:
👍